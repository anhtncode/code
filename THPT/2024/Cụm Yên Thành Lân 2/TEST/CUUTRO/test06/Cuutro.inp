141 567
334 500 169 724 478 358 962 464 705 145 281 827 961 491 995 942 827 436 391 604 902 153 292 382 421 716 718 895 447 726 771 538 869 912 667 299 35 894 703 811 322 333 673 664 141 711 253 868 547 644 662 757 37 859 723 741 529 778 316 35 190 842 288 106 40 942 264 648 446 805 890 729 370 350 6 101 393 548 629 623 84 954 756 840 966 376 931 308 944 439 626 323 537 538 118 82 929 541 833 115 639 658 704 930 977 306 673 386 21 745 924 72 270 829 777 573 97 512 986 290 161 636 355 767 655 574 31 52 350 150 941 724 966 430 107 191 7 337 457 287 753 
90 92 910
61 139 222
117 131 947
11 41 414
122 129 592
9 80 411
14 79 538
116 124 596
93 95 351
139 139 375
22 96 22
135 138 669
91 122 735
53 116 419
20 23 789
79 112 729
88 91 484
41 62 311
131 134 515
68 92 936
108 116 250
22 43 799
129 137 9
63 68 990
131 136 486
132 138 524
36 55 504
116 116 459
132 138 797
134 134 590
71 80 158
27 51 539
25 31 180
1 118 959
128 131 889
121 132 203
96 113 56
24 89 363
92 102 434
118 135 845
6 17 999
29 88 22
59 90 477
115 116 76
137 137 511
135 136 862
24 87 790
27 34 3
10 88 286
16 67 618
69 109 933
80 99 722
128 136 330
112 123 426
121 135 550
109 117 146
12 58 754
65 100 280
52 96 530
130 132 867
118 127 196
54 112 287
31 99 283
47 119 115
139 139 672
5 28 314
103 128 54
66 74 833
77 102 757
121 139 647
64 83 145
72 77 130
46 71 451
34 121 45
86 88 440
51 95 155
41 86 650
73 90 475
104 126 19
34 95 959
59 93 626
110 122 315
21 75 875
120 140 834
11 88 298
45 62 774
122 137 669
131 136 103
20 72 628
8 117 528
87 118 925
25 69 62
81 124 433
21 134 726
59 105 143
20 88 65
4 11 361
135 137 271
124 138 834
112 120 897
14 60 551
140 140 696
51 80 126
68 81 659
76 92 467
25 114 852
104 123 465
110 136 801
39 91 927
127 138 171
11 97 228
58 74 165
33 139 87
96 118 578
92 120 628
130 136 424
38 39 963
123 131 738
7 18 526
136 136 203
79 87 327
106 112 412
26 134 521
39 71 189
49 113 852
50 138 901
88 125 579
46 133 478
72 117 440
47 117 358
68 121 109
104 117 802
8 104 429
21 45 406
54 106 705
4 28 73
22 43 486
111 137 627
51 117 358
9 48 870
33 77 23
7 69 718
84 98 42
30 72 230
53 100 933
18 107 54
42 62 735
27 127 458
128 131 964
69 104 912
73 76 849
22 108 224
5 95 512
40 116 460
59 82 871
75 131 206
58 87 399
1 22 194
44 129 535
57 118 177
43 47 549
89 129 414
3 124 856
29 67 612
139 139 679
60 116 297
124 134 314
29 53 819
46 113 933
53 124 170
2 26 489
62 113 498
51 121 146
36 135 652
91 135 259
53 100 193
132 133 182
21 130 776
22 40 998
65 133 562
43 140 542
44 50 814
128 137 78
52 102 214
22 23 602
127 136 671
61 88 85
64 139 499
13 28 848
33 104 222
103 109 364
127 136 490
51 65 543
5 110 592
77 87 233
45 98 976
129 132 423
89 104 585
112 125 865
71 86 546
89 111 679
77 107 520
26 140 945
45 57 246
128 132 871
13 48 133
79 137 88
88 118 902
42 110 528
38 124 16
66 94 544
108 117 944
88 131 464
114 124 589
62 117 61
71 79 955
20 130 691
53 136 431
99 133 68
24 98 36
78 80 39
124 134 225
97 132 360
109 139 945
9 95 727
31 46 356
122 137 497
113 136 965
77 88 914
68 93 949
115 131 607
88 137 405
78 103 813
58 105 566
103 133 142
76 90 257
25 65 839
37 40 16
3 17 842
102 124 638
97 119 651
22 117 894
60 92 453
49 135 234
96 109 635
77 125 149
126 128 214
37 41 201
71 109 859
80 140 362
36 74 677
33 50 903
36 70 949
13 98 675
105 107 924
24 88 879
96 114 620
1 44 946
52 123 393
114 128 699
90 113 939
2 18 462
30 106 962
11 134 516
30 103 870
58 67 972
38 117 216
21 36 331
78 113 289
23 89 86
124 128 775
61 116 952
55 110 680
49 95 74
83 106 133
64 109 114
138 140 791
31 54 29
86 114 201
136 136 975
84 131 868
68 128 169
98 99 630
27 117 310
40 80 81
105 114 668
118 140 865
93 98 827
48 124 727
43 122 682
83 89 687
99 127 722
1 50 416
136 139 125
112 116 763
82 119 807
1 79 603
41 48 482
25 49 631
3 40 85
9 23 294
104 137 55
106 114 249
18 77 132
81 140 959
47 47 996
65 102 480
49 68 919
9 112 499
4 102 463
125 139 23
18 83 38
118 139 459
116 123 99
53 129 601
87 96 353
3 85 541
115 140 763
120 133 398
116 138 829
33 65 231
78 119 659
88 122 941
91 138 163
104 133 793
83 111 399
22 49 947
38 140 639
27 54 516
100 124 616
83 93 571
59 93 558
57 106 525
0 126 331
57 99 334
83 89 738
121 138 279
126 127 715
105 124 950
100 104 338
134 140 624
41 111 609
112 137 86
128 132 653
16 17 827
55 59 250
117 131 820
97 104 942
126 130 566
139 140 217
7 14 610
66 82 894
0 114 301
96 129 590
48 65 969
4 5 269
109 139 423
36 100 911
5 107 392
126 130 875
95 139 256
89 95 973
31 83 548
4 66 606
21 135 951
128 128 482
25 97 440
33 113 763
45 69 416
75 138 760
67 91 487
89 111 946
16 57 701
87 121 788
50 121 975
63 92 284
137 140 377
101 129 724
75 115 777
55 120 183
27 137 926
94 94 678
36 129 535
26 39 69
136 137 784
49 140 427
79 112 613
91 118 409
46 54 186
64 123 444
22 32 851
30 32 651
90 113 455
77 118 466
104 131 794
22 28 973
129 130 443
101 138 771
138 138 382
93 132 238
127 134 596
6 38 43
8 127 59
30 82 671
98 116 259
22 124 784
136 138 611
101 109 435
60 123 504
62 121 939
48 98 751
132 136 459
123 140 757
80 106 490
118 135 76
113 127 834
41 71 49
45 65 434
11 17 736
70 96 838
37 91 329
24 44 245
80 100 253
85 101 536
17 57 498
119 136 16
51 60 101
72 84 524
111 135 634
140 140 855
120 134 920
14 113 932
114 133 488
53 138 484
39 85 194
52 54 561
42 100 498
106 120 541
52 96 393
97 138 579
48 50 278
73 138 621
65 135 277
50 80 579
20 110 490
18 23 925
48 105 9
49 137 478
32 33 608
96 110 723
62 121 762
58 83 164
45 129 933
72 122 955
45 123 397
91 106 905
61 117 681
33 65 388
13 85 518
86 121 242
129 130 524
33 42 622
34 54 917
27 59 580
101 118 751
48 62 82
34 37 677
117 136 785
55 138 609
109 120 930
41 109 56
52 62 923
121 132 987
29 123 578
9 118 917
93 140 470
70 122 694
67 114 950
67 135 53
95 136 488
65 107 956
59 89 181
23 138 66
116 127 579
69 99 141
99 126 446
29 100 490
75 86 334
47 77 283
76 78 897
11 63 883
85 86 232
72 84 480
117 127 352
130 138 647
139 140 190
107 131 487
71 139 615
66 69 778
46 104 862
12 69 456
97 115 753
29 77 282
77 84 948
52 125 136
96 115 860
41 115 254
13 24 644
124 134 233
94 104 927
16 91 802
4 92 856
137 137 574
118 133 474
74 83 354
53 116 700
62 140 466
122 124 982
138 138 382
69 103 672
63 65 33
41 61 166
62 82 294
4 4 949
91 112 167
12 68 21
6 10 789
76 120 970
73 134 16
129 129 169
140 140 98
59 95 200
82 113 200
12 82 272
21 76 86
36 46 332
76 78 445
62 112 361
61 135 432
102 109 269
40 129 338
9 57 678
124 140 23
96 101 543
53 83 389
23 96 648
79 90 585
130 133 494
39 54 483
119 120 754
108 111 96
130 134 965
67 129 107
131 134 594
8 128 869
120 120 969
8 92 80
4 56 532
102 130 664
133 138 937
121 126 875
106 131 681
93 116 726
123 127 560
132 136 833
9 21 376
130 135 963
