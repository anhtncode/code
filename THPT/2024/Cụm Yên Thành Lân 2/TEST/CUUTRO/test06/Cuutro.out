3452 4569 6562 11226 11841 13283 15741 17208 22221 21947 24369 26057 29009 30560 30454 33349 33732 31086 30053 32929 37378 43833 44578 44371 46770 48600 51445 51622 51769 55432 56693 56888 58956 60989 60741 64933 64884 66900 68459 68414 69403 70560 72051 72106 74568 77642 78248 80134 81312 82286 81322 84028 87628 88623 87517 87031 87323 87715 89555 89698 91573 95440 93368 93505 95994 96508 96821 97825 98351 97812 100081 101760 102504 102838 102896 105152 108496 109601 110682 110407 109866 111084 110412 109768 108710 107750 108609 110012 111256 111000 112031 109761 108601 107503 106605 107223 108571 108404 106625 104751 104398 104909 103167 103966 101338 101832 98966 97123 98978 97153 95580 95036 93965 90383 87936 87650 84717 82309 77469 77662 75202 74411 71863 71373 63927 64487 64201 64014 62618 61240 59190 53303 49880 46479 41227 38083 30039 25866 16103 11555 121 